openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/penalites.py

```diff
@@ -1,92 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
@benjello : Do you know what this file is for ? It was not included in the model.